src/Promenade.Web/Views/Home/LocationDetails.cshtml

```diff
@@ -82,8 +107,8 @@
                                                 <a href="/Locations/@location.Stub">@location.Name</a>
```

Comment:
This should use `@Url.Action()` to construct the URL.

Reply:
Fixed